.circleci/config.yml

```diff
@@ -9,26 +9,25 @@ orbs:
 jobs:
   "python-3_7":
     docker:
-      - image: circleci/python:3.7
+      - image: cimg/python:3.7
```

Comment:
The pattern is repeated, so I guess there should only be one job definition that takes parameters (py version, requirements file).  But something for a separate PR later.